### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by techamigo**

### CODE REVIEW

File: docs/deployment-guide/planning-guidance.md [hunk 1]
```diff
@@ -3,4 +3,36 @@ sidebar_position: 3
...
+| 
+| Route53 Hosted Zone | $0.50 per month | Delgated zone for Diego to manage DNS records for tooling and customer applications | https://aws.amazon.com/route53/pricing/ |
+
+All the above AWS services are mandatory to run Diego. 
+
+## Diego licencing costs 
+
+Diego software is free to use. 
+
+We also provide additional onboardinig service for a smooter, faster setup of customer applications:
```
Comment (amigos-sian) [inline docs/deployment-guide/planning-guidance.md:28]: some corrections -> We also provide additional on-boarding service for a smoother, faster set-up of customer applications:

File: docs/deployment-guide/planning-guidance.md [hunk 2]
```diff
@@ -3,4 +3,36 @@ sidebar_position: 3
...
+| ------------ | ------------ | ------------------------ | ------------------------ |
+| EKS control plane | $0.10 | Cluster to host Diego tooling and customer applications | https://aws.amazon.com/eks/pricing/ |
+| EC2 - T3.large | $0.08 | The compute for the EKS cluster. Diego provisions single node and sets up horizontal elastic auto-scaling based on demand  | https://aws.amazon.com/ec2/pricing/on-demand/ |
+| EBS - gp2 | $0.12 Gb/month | The represent root disk for cluster nodes | https://aws.amazon.com/ec2/pricing/on-demand/ |
+| NAT Gateway | $0.05 | Provide egress from the cluster | https://aws.amazon.com/vpc/pricing/ |   
+| ELB | $0.27 | Provides ingress for Diego tooling and customeer applications | https://aws.amazon.com/elasticloadbalancing/pricing/?nc=sn&loc=3 |
+| 
+| Route53 Hosted Zone | $0.50 per month | Delgated zone for Diego to manage DNS records for tooling and customer applications | https://aws.amazon.com/route53/pricing/ |
+
+All the above AWS services are mandatory to run Diego.
```
Comment (amigos-sian) [inline docs/deployment-guide/planning-guidance.md:22]: maybe "to run" -> "for running"

File: docs/deployment-guide/planning-guidance.md [hunk 3]
```diff
@@ -3,4 +3,36 @@ sidebar_position: 3
...
+| EC2 - T3.large | $0.08 | The compute for the EKS cluster. Diego provisions single node and sets up horizontal elastic auto-scaling based on demand  | https://aws.amazon.com/ec2/pricing/on-demand/ |
+| EBS - gp2 | $0.12 Gb/month | The represent root disk for cluster nodes | https://aws.amazon.com/ec2/pricing/on-demand/ |
+| NAT Gateway | $0.05 | Provide egress from the cluster | https://aws.amazon.com/vpc/pricing/ |   
+| ELB | $0.27 | Provides ingress for Diego tooling and customeer applications | https://aws.amazon.com/elasticloadbalancing/pricing/?nc=sn&loc=3 |
+| 
+| Route53 Hosted Zone | $0.50 per month | Delgated zone for Diego to manage DNS records for tooling and customer applications | https://aws.amazon.com/route53/pricing/ |
+
+All the above AWS services are mandatory to run Diego. 
+
+## Diego licencing costs
```
Comment (amigos-sian) [inline docs/deployment-guide/planning-guidance.md:24]: licencing -> licensing

File: docs/deployment-guide/planning-guidance.md [hunk 4]
```diff
@@ -3,4 +3,36 @@ sidebar_position: 3
...
+| NAT Gateway | $0.05 | Provide egress from the cluster | https://aws.amazon.com/vpc/pricing/ |   
+| ELB | $0.27 | Provides ingress for Diego tooling and customeer applications | https://aws.amazon.com/elasticloadbalancing/pricing/?nc=sn&loc=3 |
+| 
+| Route53 Hosted Zone | $0.50 per month | Delgated zone for Diego to manage DNS records for tooling and customer applications | https://aws.amazon.com/route53/pricing/ |
+
+All the above AWS services are mandatory to run Diego. 
+
+## Diego licencing costs 
+
+Diego software is free to use.
```
Comment (amigos-sian) [inline docs/deployment-guide/planning-guidance.md:26]: People would be thinking - "does this mean it's open source?" so is there a way we can provide clarity on that here?

**Review by amigos-sian — CHANGES_REQUESTED** [review]

File: docs/deployment-guide/planning-guidance.md (hunk 4 above)
Comment (techamigo) [reply docs/deployment-guide/planning-guidance.md:26]: no really, we are not providing source to them, its just the software (i.e executable) that we are saying is free to use

**Pull request merged**